packages/graphback-codegen-client/src/ClientCRUDPlugin.ts

```diff
@@ -41,28 +41,27 @@ export const CLIENT_CRUD_PLUGIN = "ClientCRUDPlugin";
  */
 export class ClientCRUDPlugin extends GraphbackPlugin {
```

Comment:
I find the naming of the functions a bit confusing for this plugin.

`init` => this doesn't reflect what's happening. I believe `getDocuments` would be a better name as it is doing: 

1. Getting the models from the schema.
2. Generate documents for those models.
3. Returning documents.

`generateResources` should be called `createResources`. This function is delegating the docment generation and executes the persistence of the documents to the filesystem. I believe `createResources` is more clear.

`createDocuments` should be called `generateDocuments`. It is doing the generation.